games/TicTacToe/project/contracts/tictactoe-contract/tests/utils/setup.rs

```diff
@@ -14,18 +14,18 @@ abigen!(Contract(
 const TICTACTOE_CONTRACT_BINARY_PATH: &str = "./out/debug/tictactoe-contract.bin";
 const TICTACTOE_CONTRACT_STORAGE_PATH: &str = "./out/debug/tictactoe-contract-storage_slots.json";
 
-pub struct Player {
-    pub contract: TicTacToe,
-    pub identity: Identity,
+pub(crate) struct Player {
+    pub(crate) contract: TicTacToe<WalletUnlocked>,
+    pub(crate) identity: Identity,
 }
 
-pub async fn setup() -> (Player, Player) {
-    let num_wallets = 2;
+pub(crate) async fn setup() -> (Player, Player) {
```

Comment:
Out of curiosity, is the need for `(crate)` related to the SDK or is it general Rust?

Reply:
Rust, restricting access only to within the crate